src/Brick/Widgets/FileBrowser.hs

```diff
@@ -779,5 +779,10 @@ fileTypeMatch tys i = maybe False (`elem` tys) $ fileInfoFileType i
 -- @TEST.XML@ and it will match directories regardless of name.
 fileExtensionMatch :: String -> FileInfo -> Bool
 fileExtensionMatch ext i = case fileInfoFileType i of
+    Just Directory -> True
     Just RegularFile -> ('.' : (toLower <$> ext)) `isSuffixOf` (toLower <$> fileInfoFilename i)
-    _ -> True
+    Just SymbolicLink -> case fileInfoLinkTargetType i of
+        Nothing -> error "impossible"
```

Comment:
I'm not entirely sure this case is impossible. It could be that the link destination could not be read due to permissions issues, so I think this case should result in `False`.

Reply:
Oh okay I didn't think of that. It should be correct after that second commit I did then.

Reply:
Ah, thanks - I didn't look ahead to the next commit. Thanks!